WordPress/wordpress-playground
# [CLI] Add WordPress constants using CLI arguments

## Motivation for the change, related issues

I was testing #1789 and added support for defining constants to the CLI during development. 

If feels like a useful feature, so I want to include it in the CLI.

## Implementation details

A new `--constants` argument enables users to add WordPress constants to Playground. 

## Testing Instructions (or ideally a Blueprint)

- Run Playground `bun ./packages/playground/cli/src/cli.ts server --constants PLAYGROUND_FORCE_AUTO_LOGIN_ENABLED=true`
- [Open this URL](http://127.0.0.1:9400/?playground_force_auto_login_as_user=admin)
- Confirm that you are logged in
